app/(tabs)/(touren)/stagesMapViewWrapper.tsx

```diff
@@ -8,7 +8,16 @@ import renderBackAction from "@/components/TopNavigation/renderBackAction";
 type localParams = {
   tourId?: string;
 };
-
+/**
```

Comment:
```suggestion

/**
```
Ein Zeilenumbruch noch